glusterd2/commands/volumes/volume-option.go

```diff
@@ -113,13 +134,25 @@ func xlatorAction(c transaction.TxnCtx, txnOp txnOpType, volOp volumeOpType) err
 			reqOptions[key] = op.DefaultValue
 		}
 
-	}
-	var volinfo volume.Volinfo
-	if err := c.Get("volinfo", &volinfo); err != nil {
-		return err
+	case xlator.VolumeStart:
+		fallthrough
+	case xlator.VolumeStop:
+		//start the daemon, if daemon default values is "on",
+		//or if particular  daemon  options are set to "on" in volume option
```

Comment:
`option` (singular)

there might be extra spaces between words

Reply:
done